### PR TITLE
Bug 1223668 - Remote control client page preference

### PR DESCRIPTION
**Pull request by MDTsai**
Store remote control client page Gecko preference in Gaia

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=374fee0ec0a4a78d390caca9f39806179ac86a4a

File: build/config/tv/custom-prefs.js [hunk 1]
```diff
@@ -12,3 +12,7 @@ user_pref('dom.meta-viewport.enabled', false);
 user_pref('dom.presentation.enabled', true);
 user_pref('devtools.useragent.device_type', 'TV');
 user_pref('dom.apps.customization.enabled', false);
+
+// Remote Control URL, for static file
```
Comment (rexboy7) [inline build/config/tv/custom-prefs.js]: "Remote control URL. Gecko loads its required static file in this app."

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=709dd60282ead1f091e87be882576d4c31b67049

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=c7f809550776ae063ea5804d0d76eddb4ad1718d

**Pull request merged**